FoodTracker/Controllers/MealViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  MealViewController.swift
+//  FoodTracker
+//
+//  Created by BrainX IOS 4 on 2020-12-14.
+//
+import UIKit
+import os.log
+
+class MealViewController: UIViewController{
+
+    //MARK: -Outlets
+    @IBOutlet weak var tfMealName: UITextField!
+    @IBOutlet weak var imgMeal: UIImageView!
+    @IBOutlet weak var ratingControl: RatingControl!
+    @IBOutlet weak var saveButton: UIBarButtonItem!
+    
+    //MARK: -Properties
+    var meal: Meal?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tfMealName.delegate = self
+        setMeal()
+        updateSavedButtonStatus()
+    }
+    
+    //MARK: -Actions
+    @IBAction func selectImageFromGallery(_ sender: UITapGestureRecognizer) {
+        
+        tfMealName.resignFirstResponder()
+        let imagePickerController = UIImagePickerController()
+        imagePickerController.sourceType = .photoLibrary
+        imagePickerController.delegate = self
+        present(imagePickerController, animated: true, completion: nil)
+    }
+    
+    //MARK: -Private Methods
+    private func updateSavedButtonStatus(){
```

Comment:
selectImageFromGallery method has a after the parameters closing bracket")" and the methods opening bracket"{" and that looks good to me I also put space there. But the updateSavedButtonStatus don't have that space and thats the inconsistency in the code styling.
This is alos applicable for if and else block e.g.
if isTrue { 
} else {
}
Observe the spaces.